src/Compilers/Core/Portable/Symbols/Attributes/CommonAssemblyWellKnownAttributeData.cs

```diff
@@ -443,5 +444,27 @@ public HashSet<TNamedTypeSymbol> ForwardedTypes
             }
         }
         #endregion
+
+        #region ExperimentalAttribute
+        private ObsoleteAttributeData _obsoleteAttributeData = ObsoleteAttributeData.Uninitialized;
+        public ObsoleteAttributeData ObsoleteAttributeData
```

Comment:
>ObsoleteAttributeData [](http://example.com/codeflow?start=37&length=21)

Should the property be named `ExperimentalAttributeData` if this is intended specifically for that attribute?